v2/devices/mimameid.yml

```diff
@@ -57,6 +57,11 @@ operating_systems:
                   checksum:
                     sum: "c0216239cc6dc11e23940884d0f6d1183b29898e63b561da8122bf69636f6c7c"
                     algorithm: "sha256"
+                # TODO: tells Volla to ship a vbmeta.img as part of the bootstrap zip.
```

Comment:
[`volla-mimameid-11.0-ubports-installer-bootstrap.zip.new`](https://volla.tech/filedump/volla-mimameid-11.0-ubports-installer-bootstrap.zip.new) was uploaded containing the `vbmeta.img` for testing, it can be renamed at the same time as this is merged assuming all is good with it :)

Reply:
Thank you for proactively including that. I've tested the new bootstrap zip, and it seems to work.

That said, I think it's better to rename it to e.g. `volla-mimameid-11.0-ubports-installer-bootstrap-v2.zip` (that's what I have in the PR at the moment, can be changed if you prefer otherwise). That way, you can rename the file now, and users will see no "gap period" where one cannot flash because the checksum mismatch. Then, after a day or so (depending on the caching of the web server, maybe), you can drop the old file.

Reply:
Renamed it to `volla-mimameid-11.0-ubports-installer-bootstrap-v2.zip` as suggested